core/src/main/java/org/apache/accumulo/core/spi/balancer/SimpleLoadBalancer.java

```diff
@@ -297,10 +275,7 @@ List<TabletMigration> move(ServerCounts tooMuch, ServerCounts tooLittle, int cou
        * is only one tabletserver that holds all of the tablets. Here we check to see if in fact
        * that is the case and if so set the value to 0.
        */
-      tooLittleCount = tooLittleMap.get(table);
-      if (tooLittleCount == null) {
-        tooLittleCount = 0;
-      }
+      Integer tooLittleCount = tooLittleMap.getOrDefault(table, 0);
```

Comment:
I briefly looked, and didn't see an Integer.ZERO. Such a thing would be useful here, if it existed, so we don't have to keep boxing `0`. But, I'm sure this is fine. The JRE can optimize stuff like this if it wanted to.